docs/external-dns.md

```diff
@@ -49,7 +71,7 @@ development environment.
 # contains external-dns config
 
 # allows to delete the dns registry when deleting a service
-policy: sync 
+policy: upsert-only
```

Comment:
What this change really does?

Reply:
Good question, this is determines the behaviour of the external-dns.

The value `sync` will no only create DNS records when a service is deployed but it will also delete the record when the service is terminated.

While the `upsert-only` value will only create/update the DNS records and will not delete any records